src/test/regress/input/columnar_chunk_filtering.source

```diff
@@ -101,3 +101,13 @@ EXPLAIN (analyze on, costs off, timing off, summary off)
   SELECT count(*) FROM multi_column_chunk_filtering WHERE a > 50000 AND b > 50000;
 
 DROP TABLE multi_column_chunk_filtering;
+
+--
+-- https://github.com/citusdata/citus/issues/4780
+--
+create table part_table (id int) partition by range (id);
+create table part_1_row partition of part_table for values from (150000) to (160000);
+create table part_2_columnar partition of part_table for values from (0) to (150000) using columnar;
```

Comment:
should we also test with more than one columnar partitions where we prune chunks from different partitions in the same query ?

row_part -> 0, 99 (actually not needed at allt)
columnar_part_1 -> 100, 79999
columnar_part_2 -> 80000, 119999

```sql
select filtered_row_count('select count(*) from part_table where id > 50000 and id < 110000');
```

Reply:
The filtered_row_count function only returns a single number, so it's not really sufficient to make this test change. I'll just leave it as-is matching the issue in the tracker. We can refactor some other things later.